packages/instantsearch-ui-components/src/components/TrendingItems.tsx

```diff
@@ -90,7 +90,6 @@ export function createTrendingItemsComponent({
 
         <Layout
           classNames={cssClasses}
-          translations={translations}
```

Comment:
does this mean the "previous" and "next" labels aren't customisable, or is that done another way now?

Reply:
The translations we passed were the ones of the widget, and it didn't make sense to do so. Layouts have their own translations which they receive as a prop.